scrape/server/flask_app/repo/users.py

```diff
@@ -10,7 +13,20 @@ class User:
 
     def __init__(self, config: Config):
         self.postgres_uri = config.POSTGRES_URL
-        self.postgres = Postgres(config.POSTGRES_URL)
+        if config.POSTGRES_URL == "":
```

Comment:
On second thought... instead of using an if statement here, we can set the default value for `config.POSTGRES_URL`  when we're attempting to create `Configuration` to be localhost of Postgres to simplify and clean things up a bit

Reply:
Merged [#13 ](https://github.com/Matthew-Luk/Projects/pull/13) to address this issue